Bottom: b3e029f05a17fe5daa53be00c39104f09019e81d
Top:    8dce33df924ac151c520dc718ffec5517ae93c40
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-05-23 10:51:51 +0800

vhost: bypass wrap counter in SET/GET_VRING_BASE

Signed-off-by: Jason Wang <jasowang@redhat.com>


---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 54490cc..b37f42e 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -1412,10 +1412,12 @@ long vhost_vring_ioctl(struct vhost_dev *d, unsigned int ioctl, void __user *arg
 			break;
 		}
 		vq->last_avail_idx = s.num;
+#if 0
 		if (vhost_has_feature(vq, VIRTIO_F_RING_PACKED)) {
 			vq->last_avail_wrap_counter = s.num >> 31;
 			vq->avail_wrap_counter = vq->last_avail_wrap_counter;
 		}
+#endif
 		/* Forget the cached index value. */
 		vq->avail_idx = vq->last_avail_idx;
 		break;
